daq_main2.py

```diff
@@ -28,12 +28,9 @@
 logging.getLogger('ophyd').setLevel(logging.WARN)
 logging.getLogger('caproto').setLevel(logging.WARN)
 handler1 = handlers.RotatingFileHandler('lsdcServerLog.txt', maxBytes=5000000, backupCount=100)
-handler2 = handlers.RotatingFileHandler('/var/log/dama/%slsdcServerLog.txt' % os.environ['BEAMLINE_ID'], maxBytes=5000000, backupCount=100)
```

Comment:
This change was necessary to fix an issue that came up while trying to meet the compliance requirements, but was not one of those requirements. I will restore this once I am able to resume beam testing and ensure it is working again.